src/fileCollectionFromFileList.ts

```diff
@@ -0,0 +1,20 @@
+import { FileCollection } from './FileCollection';
+
+/**
+ * Generate a FileCollection from a directory path
+ * By default this method will expand all zip and gzip files
```

Comment:
Better you checked this PR ... I reworked the package so that all the 'from' will automatically expand the zip and gzip (even if coming from weservice)